src/main/java/org/folio/roles/service/capability/ApiRoleCapabilityService.java

```diff
@@ -49,7 +49,6 @@ public void update(UUID roleId, List<UUID> capabilityIds) {
   @Override
   public void update(UUID roleId, CapabilitiesUpdateRequest request) {
     checkRoleIsNotDefault(roleId);
-    verifyCapabilities(request.getCapabilityIds(), request.getCapabilityNames());
```

Comment:
we remained validation for POST request, because previously before our changes validation for empty capabilities was exists